eng/Directory.Build.Common.props

```diff
@@ -100,12 +100,6 @@
     </NoWarn>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(InheritDocEnabled)' != 'false' and '$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
TargetFramework is only available after csproj evaluation so should be consumed in targets.